### PR TITLE
New Carrot native format with third party pixel

### PR DESCRIPTION
**Pull request by ioanna0**
Creates a new exactly the same carrot native format with an additional Tracking Pixel:
` `

Fluid • Creative template
Style name: Carrot - 3rd party tracking

### CODE REVIEW

File: src/carrot-third-party/web/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <img class="thumbnail__img" src="[%TrailImage%]" aria-hidden="true">
+  </div>
+  <div class="body margin">
+    <h1 class="body__title">[%Headline%]</h1>
+    <p class="body__standfirst">[%Standfirst%]</p>
+    <p class="body__tagline">Paid for by
+      <img class="logo__img" src="[%BrandLogo%]" aria-hidden="true"></p>
+  </div>
+  <a class="u-faux-block-link__overlay" href="%%CLICK_URL_UNESC%%%%DEST_URL%%" target="_top"></a>
+  <img src="[%TrackingPixel%]%%CACHEBUSTER%%" class="creative__pixel">
```
Comment (ioanna0) [inline src/carrot-third-party/web/index.html]: Comparing with the existing carrot native ad, line `13` is the new change

File: src/carrot-third-party/test.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (adamnfish) [inline src/carrot-third-party/test.json]: Should this test data include a tracking pixel?

Comment (ioanna0) [reply src/carrot-third-party/test.json]: Yes good point, have added one now

**Comment by adamnfish** [comment]
I agree this is the same apart from the tracking pixel! Assuming that's correct (I'm not sure of the requirements here), LGTM 👍 

```diff
diff -r carrot/test.json carrot-third-party/test.json
6c6,7
<   "BrandLogo": "https://seeklogo.com/images/A/airbnb-logo-7F4086530F-seeklogo.com.png"
---
>   "BrandLogo": "https://seeklogo.com/images/A/airbnb-logo-7F4086530F-seeklogo.com.png",
>   "TrackingPixel": "https://gu.com/tracking-pixel.gif"
diff -r carrot/web/index.html carrot-third-party/web/index.html
12a13
>   <img src="[%TrackingPixel%]%%CACHEBUSTER%%" class="creative__pixel">
```

**Pull request merged**